unittest/CMakeLists.txt

```diff
@@ -1,13 +1,6 @@
-if(NOT pinocchio_FOUND)
-  message(
-    FATAL_ERROR
-      "Pinocchio is required to build unit tests\n"
-      "please either install Pinocchio or reconfigure without unit tests (-DBUILD_TESTING=OFF)"
-  )
-endif(NOT pinocchio_FOUND)
```

Comment:
Same comment here.

Reply:
The only test is a Python test. Then, checking if find_package(pinocchio) have succeed to detect if Pinocchio Python binding will not help.

More generally, I don't think that CMake should check for Python dependencies with CMake modules. Python dependencies should be managed by a package manager (pip, conda, ...).

Reply:
Hi @jorisv!

This is the last comments. Please ping us when it is done! Thanks!

Reply:
Could you provide details on why those cases fail? In any case, I'll let @nim65s decide this. He is more competent in judging your suggestion. Thanks!

Reply:
We only have python unittest, and when python is active, there is a `add_project_dependency(pinocchio 2.7.0 REQUIRED)`, so the only case where things could fail would be if someone installs pinocchio without python interface and try to run example-robot-data python unittest.
This case can be dismissed, so I think we can remove this error message.